misc/db_schema.yaml

```diff
@@ -426,7 +427,7 @@ devices:
     sysContact: { Field: sysContact, Type: text, 'Null': true, Default: null, Extra: '' }
     sysDescr: { Field: sysDescr, Type: text, 'Null': true, Default: null, Extra: '' }
     sysName: { Field: sysName, Type: varchar(128), 'Null': true, Default: null, Extra: '' }
-    sysObjectID: { Field: sysObjectID, Type: varchar(128), 'Null': true, Default: null, Extra: '' }
+    sysObjectID: { Field: sysObjectID, Type: varchar(64), 'Null': true, Default: null, Extra: '' }
```

Comment:
It looks like your database schema is not up to date, please update, run validate.php and fix the issues present then rebuild db_schema.yaml